config/jobs/kubernetes/sig-node/dra.jinja

```diff
@@ -159,8 +166,8 @@
         resources:
           limits:
             cpu: 2
-            memory: 9Gi
+            memory: 6Gi
           requests:
             cpu: 2
-            memory: 9Gi
+            memory: 6Gi
```

Comment:
I see, you only change memory amount for all types of jobs. Why not to do it only for canary jobs as well? We'll anyway need one more iteration for the rest of the changes, right?

Reply:
We could, but doing it for the presubmits wouldn't tell us much for the periodics.

This seemed safe enough that it could be done directly, without a trial run in canary first - I'm fine either way, up to you.

Reply:
I've pushed a version where the memory limits only get changed in the canary jobs.

Reply:
/lgtm
/approve